test/translator/status_test.cc

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 Google LLC//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "lib/translator/status.h"
+
+#include "absl/base/casts.h"
+#include "gtest/gtest.h"
+
+// Tests
+
+namespace {
+
+TEST(TranslateGenericCommandStatus, ShouldReturnCorrectStatus) {
+  uint8_t sct = 0x0;  // kGeneric
+  uint8_t sc = 0x0;   // kSuccess;
+  translator::ScsiStatus result;
```

Comment:
Since we rely on zero-init construction at line 103 in status.cc, this might need to be translator::ScsiStatus result = {}.

Reply:
I'm changing status.cc to not require zero initialization. Instead, status.cc will explicitly set `ScsiStatus` to the "default" type of kCheckCondition and kNoSense when there's no translation available